ui/app/src/Main.elm

```diff
@@ -53,8 +59,32 @@ init location =
 
                 _ ->
                     nullFilter
+
+        apiUrl =
+            flags
```

Comment:
I need a way to differentiate between using the template and using elm-reactor. It could be as simple as a boolean that says `development=true`, but if we want to use elm-reactor (which I think we should) we need a way to differentiate